features/promotion/receiving_percentage_discount_on_products_from_specific_taxon.feature

```diff
@@ -0,0 +1,51 @@
+@promotion
+Feature: Receiving percentage discount on products from specific taxon
+    In order to pay less while buying goods from promoted taxons
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And store classifies its products as "T-Shirts", "Mugs" and "Books"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And it belongs to "T-Shirts"
+        And the store has a product "PHP Mug" priced at "€20.00"
+        And it belongs to "Mugs"
+        And there is a promotion "T-Shirts promotion"
+        And it gives "20%" percentage discount to every order item containing products from "T-Shirts" taxon
+
+    @todo
+    Scenario: Receiving percentage discount on t-shirt
+        When I add product "PHP T-Shirt" to the cart
+        Then it should be discounted at "-€20.00"
+        And my cart total should be "€80.00"
+        And my discount should be "-€20.00"
+
+    @todo
+    Scenario: Receiving percentage discount on multiple t-shirts
+        When I add 3 products "PHP T-Shirt" to the cart
+        Then they should be discounted at "-€60.00"
+        And my cart total should be "€240.00"
+        And my discount should be "-€60.00"
+
+    @todo
+    Scenario: Receiving percentage discount does not affect other categories
+        When I add product "PHP T-Shirt" to the cart
+        And I add product "PHP Mug" to the cart
+        Then product "PHP T-Shirt" should be discounted at "-€20.00"
+        And product "PHP Mug" should not be discounted
+        And my cart total should be "€100.00"
+        And my discount should be "-€20.00"
+
+    @todo
+    Scenario: Receiving different discounts on different categories
+        Given the store has a product "PHP Book" priced at "€50.00"
+        And it belongs to "Books"
```

Comment:
Why creating another product? Can't we simply create a promotion for Mugs and remove these two steps?